seqr/views/apis/staff_api_tests.py

```diff
@@ -260,6 +265,261 @@
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'5', 'subject_id': u'NA20874', 'entity:family_id': u'NA20874',
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'6', 'subject_id': u'NA20875', 'entity:family_id': u'NA20875',
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'7', 'subject_id': u'NA20876', 'entity:family_id': u'NA20876',
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'8', 'subject_id': u'NA20877', 'entity:family_id': u'NA20877',
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'9', 'subject_id': u'NA20878', 'entity:family_id': u'NA20878',
+          'paternal_id': '', 'maternal_id': ''},
+         {'family_id': u'10', 'subject_id': u'NA20881', 'entity:family_id': u'NA20881',
+          'paternal_id': '', 'maternal_id': ''}]],
+    [u'1kg project n\xe5me with uni\xe7\xf8de_PI_Discovery',
+     ['entity:discovery_id', 'subject_id', 'sample_id', 'Gene-1', 'Gene_Class-1',
+      'inheritance_description-1', 'Zygosity-1', 'Chrom-1', 'Pos-1', 'Ref-1', 'Alt-1', 'hgvsc-1',
+      'hgvsp-1', 'Transcript-1', 'sv_name-1', 'sv_type-1', 'significance-1'], [
+         {'Zygosity-1': 'Heterozygous', 'Pos-1': '3343353', 'Ref-1': u'GAGA', 'Alt-1': u'G',
+          'Gene-1': u'RP11-206L10.5', 'subject_id': u'NA19675_1', 'hgvsp-1': u'p.Leu126del',
+          'Gene_Class-1': 'Known', 'Transcript-1': u'ENST00000258436',
+          'hgvsc-1': u'c.375_377delTCT', 'sample_id': u'NA19675',
+          'entity:discovery_id': u'NA19675_1', 'Chrom-1': u'2',
+          'inheritance_description-1': 'de novo'},
+         {'entity:discovery_id': u'NA19678', 'subject_id': u'NA19678', 'sample_id': u'NA19678'},
+         {'entity:discovery_id': u'NA19679', 'subject_id': u'NA19679', 'sample_id': u'NA19679'},
+         {'entity:discovery_id': u'HG00731', 'subject_id': u'HG00731', 'sample_id': u'HG00731'},
+         {'entity:discovery_id': u'HG00732', 'subject_id': u'HG00732', 'sample_id': u'HG00732'},
+         {'entity:discovery_id': u'HG00733', 'subject_id': u'HG00733', 'sample_id': u'HG00733'},
+         {'entity:discovery_id': u'NA20870', 'subject_id': u'NA20870', 'sample_id': u'NA20870'},
+         {'entity:discovery_id': u'NA20872', 'subject_id': u'NA20872', 'sample_id': u'NA20872'},
+         {'entity:discovery_id': u'NA20874', 'subject_id': u'NA20874', 'sample_id': u'NA20874'},
+         {'entity:discovery_id': u'NA20875', 'subject_id': u'NA20875', 'sample_id': u'NA20875'},
+         {'entity:discovery_id': u'NA20876', 'subject_id': u'NA20876', 'sample_id': u'NA20876'},
+         {'entity:discovery_id': u'NA20877', 'subject_id': u'NA20877', 'sample_id': u'NA20877'},
+         {'entity:discovery_id': u'NA20878', 'subject_id': u'NA20878', 'sample_id': u'NA20878'},
+         {'entity:discovery_id': u'NA20881', 'subject_id': u'NA20881', 'sample_id': u'NA20881'}]
+     ]
+]
+
```

Comment:
having this much test data is very fragile. Can you test one row per file, the way you did for the discovery sheet?

Reply:
Yes. I included all the data because they are calling parameters which can only be tested as a whole. I'm going to put them into the HTTP response then I can test one row per file.

Reply:
I'm going to use `call_args` to test the data.